libevmasm/Assembly.cpp

```diff
@@ -834,6 +862,63 @@ std::map<u256, u256> const& Assembly::optimiseInternal(
 	return *m_tagReplacements;
 }
 
+namespace
+{
+uint16_t calcMaxStackHeight(std::vector<AssemblyItem> const& _items, uint16_t _args)
+{
+	uint16_t maxStackHeight = _args;
+	std::stack<size_t> worklist;
+	std::vector<int32_t> stack_heights(_items.size(), -1);
```

Comment:
```suggestion
	std::vector<size_t> stack_heights(_items.size(), std::numeric_limits<size_t>::max);
```

`stackHeight` should also be unsigned - I would assume our stack height is never going to be negative?

Reply:
Fixed